tests_integration/test_data/projects/streamlit_v2_external_access/snowflake.yml

```diff
@@ -0,0 +1,16 @@
+definition_version: 2
+entities:
+  my_streamlit:
+    type: "streamlit"
+    identifier:
+      name: test_streamlit_deploy_snowcli_ext_access
+    title: "My Fancy Streamlit"
+    stage: streamlit
+    query_warehouse: xsmall
+    main_file: streamlit_app.py
+    external_access_integrations:
+      - snowflake_docs_access_integration
+    secrets:
+      generic_secret: external_access_db.public.test_secret
```

Comment:
```suggestion
      generic_secret: snowcli_db.public.test_secret
```

Reply:
I will replace `external_access_db` with `snowcli_db` in my next PR.